randovania/server/database.py

```diff
@@ -84,10 +84,6 @@ class User(BaseModel):
     name: str = peewee.CharField()
     admin: bool = peewee.BooleanField(default=False)
 
-    @classmethod
-    def get_by_id(cls, pk) -> Self:
-        return cls.get(cls._meta.primary_key == pk)
-
```

Comment:
If that's the default implementation of peewee it might could also be removed from `MultiplayerSession`.

Reply:
It's actually a reimplementation in BaseModel, but yeah